server/pulp/server/webservices/urls.py

```diff
@@ -66,4 +67,6 @@
     url(r'^v2/plugins/types/$', TypesView.as_view(), name='plugin_types'),
     url(r'^v2/plugins/types/(?P<type_id>[^/]+)/$', TypeResourceView.as_view(),
         name='plugin_type_resource'),
+    url(r'^v2/tasks/$', TaskCollectionView.as_view(), name='task_collection'),
+    url(r'^v2/tasks/(?!search/$)(?P<task_id>[^/]+)/$', TaskResourceView.as_view(), name='task_resource'),
```

Comment:
This ugly little :golf: is a workaround. We are blocked on `/tasks/search/` and this was matching that url. When that one is implemented, we can put it above this url and remove the silliness.